src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/profiledetails/ContactPersonProfileDetailsMigrationIntTest.kt

```diff
@@ -315,19 +347,151 @@ class ContactPersonProfileDetailsMigrationIntTest(
       }
     }
 
-    private suspend fun performMigration(prisonerNumber: String? = null): String = migrationService.startMigration(ContactPersonProfileDetailsMigrationFilter(prisonerNumber))
-      .also {
-        waitUntilCompleted()
-      }.let {
-        it.migrationId
```

Comment:
The main change is to stop using the service to run migrations and use the new endpoints instead.